packages/sources/cryptocompare-test/src/endpoint/crypto.ts

```diff
@@ -1,52 +1,57 @@
-import { HttpRequestConfig, HttpResponse } from '@chainlink/external-adapter-framework/transports'
-import { PriceEndpoint, PriceEndpointParams } from '@chainlink/external-adapter-framework/adapter'
-import { BatchWarmingTransport } from '@chainlink/external-adapter-framework/transports/batch-warming'
-import { makeLogger, ProviderResult } from '@chainlink/external-adapter-framework/util'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { PriceEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { makeLogger, SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
 import {
   ProviderCryptoQuoteData,
   ProviderCryptoResponseBody,
-  CryptoEndpointTypes,
   CryptoEndpointParams,
   cryptoEndpointInputParams,
   endpoints,
 } from '../crypto-utils'
 import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
 import { customSettings, defaultEndpoint } from '../config'
-import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/routing'
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/meta'
 import { wsTransport } from './crypto-ws'
 
 const logger = makeLogger('CryptoCompare HTTP')
 
-type BatchEndpointTypes = CryptoEndpointTypes & {
+interface ProviderRequestBody {
+  fsyms: string
+  tsyms: string
+}
+
+type BatchEndpointTypes = {
+  Request: {
+    Params: CryptoEndpointParams
+  }
+  Response: SingleNumberResultResponse
+  CustomSettings: typeof customSettings
   Provider: {
-    RequestBody: never
+    RequestBody: ProviderRequestBody
```

Comment:
Is there actually a POST request body, or is it just query params?

Reply:
I think you are right. After checking this should be for POST request body, tho til now we don't send POST requests to DP. Not sure if we need this for query params, but I put a note to change this for my active PRs